src/tests/nativeaot/SmokeTests/DotnetJs/Program.cs

```diff
@@ -1,15 +1,36 @@
 using System;
+using System.Runtime.InteropServices.JavaScript;
 
-class Program
+namespace DotnetJsApp;
+
+partial class Program
 {
     static int Main(string[] args)
     {
         Console.WriteLine("Hello, DotnetJs!");
         Console.WriteLine($"Args {String.Join(", ", args)}");
 
         if (args.Length != 3 || args[0] != "A" || args[1] != "B" || args[2] != "C")
-            return 1;
+            return 11;
+            
+        var mathResult = Interop.Math(1, 2, 3);
+        Console.WriteLine($"Math result is '{mathResult}'");
+        if (mathResult != 7)
+            return 12;
 
         return 100;
     }
-}
\ No newline at end of file
+
+    static partial class Interop
+    {
+        [JSImport("interop.math", "main.js")]
+        internal static partial int Math(int a, int b, int c);
+
+        [JSExport]
+        internal static int Square(int x)
+        {
+            Console.WriteLine($"Computing square of {x}");
+            return x^2;
+        }
```

Comment:
Could you write more tests? For `JSExport`, you can take inspiration (aka copy code) from the `SharedLibrary` test - the main items of interest are EH and GC exercising methods.

For both `JSExport` and `JSImport`, it would be good to have some number of methods that exercise the actual marshaling (strings, arrays, tasks, delegates, basically,  challening cases from the table in https://learn.microsoft.com/en-us/aspnet/core/blazor/javascript-interoperability/import-export-interop?view=aspnetcore-8.0). Perhaps these tests are already written somewhere and too could be copied...

Reply:
I guess it will take some time yet before you are able to run this https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System/Runtime/InteropServices/JavaScript/JSImportExportTest.cs

Reply:
Right. We don't need super-comprehensive testing here, but something that would fail if some basic aspect of the system gets broken.

Reply:
I would rather use standard JS interop tests when we get there. I don't see value in duplicating them here. This is just a smoke test.

Also, a lot of JS interop scenarios don't work yet and are out of scope for this PR

Reply:
Strings for example don't work

Reply:
Interesting that they don't. Well, let's leave this be until they do.

Reply:
We need https://github.com/dotnet/runtime/pull/95180 for strings

Reply:
> We need [dotnet/runtime#95180](https://github.com/dotnet/runtime/pull/95180) for strings

Not in time for this PR, but I will start the next merge soon.

Reply:
It's ok, no rush on that one, I have other things to do :)